src/plugins/routing.js

```diff
@@ -49,6 +53,10 @@ function switchLocalePathFactory (i18nPath) {
   const LOCALE_CODE_KEY = '<%= options.LOCALE_CODE_KEY %>'
 
   return function switchLocalePath (locale) {
+    if(STRATEGY === STRATEGIES.NO_PREFIX && locale !== defaultLocale) {
+      console.warn('[<%= options.MODULE_NAME %>] switchLocalePath is unsupported when using no_prefix strategy', locale, defaultLocale)
```

Comment:
Same as with other warning. `switchLocalePath` is supported but only when passing current locale or not passing anything.